src/project/project-index.ts

```diff
@@ -107,6 +109,9 @@ export async function readBaseInputIndex(
         | undefined,
     markdown,
     formats,
+    draft: (firstFormat?.metadata?.[kDraft] || markdown.yaml?.[kDraft]) as
```

Comment:
> Note that this only applies to draft when set directly in document metadata (not via _metadata.yml or other merging metadata).

Is this the place where we would in the future use project-wide information to determine draft status?